tests/manage/cluster/nodes/test_nodes_restart.py

```diff
@@ -0,0 +1,200 @@
+import logging
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework.testlib import tier4, ignore_leftovers, ManageTest
+from ocs_ci.utility import aws
+from ocs_ci.ocs.cluster import CephCluster
+from tests import helpers
+
+
+logger = logging.getLogger(__name__)
+
+AWS = aws.AWS()
+
+
+@pytest.fixture()
+def instances(request):
+    """
+    Get cluster instances
+
+    Returns:
+        dict: The ID keys and the name values of the instances
+    """
+    def finalizer():
+        """
+        Make sure all instances are running
+        """
+        AWS.start_ec2_instances(instances=instances, wait=True)
+
+    request.addfinalizer(finalizer)
+
+    nodes = ocp.get_all_nodes()
+    instances = aws.get_instances_ids_and_names(nodes)
+    return instances
+
+
+@pytest.fixture()
+def resources(request):
+    """
+    Delete the resources created during the test
+
+    Returns:
+        tuple: empty lists of resources
+
+    """
+    projects = list()
+    secrets = list()
+    pools = list()
+    storageclasses = list()
+    pvcs = list()
+    pods = list()
```

Comment:
```suggestion
    projects, secrets, pools, storageclasses, pvcs, pods = [None] for i in range(6)
```

Reply:
done

Reply:
@MeridianExplorer @ebenahar Do you think, this is more readable then the original version?
If you want to squash it to one line, I would propose this:
```
    projects = secrets = pools = storageclasses = pvcs = pods = list()
```

Reply:
@dahorak it's more organized. I don't have a preference though